source/draw/Image.ooc

```diff
@@ -73,18 +73,18 @@ Image: abstract class {
 		this _canvas = null
 		super()
 	}
-	resizeWithin: func (restriction: IntSize2D) -> This {
```

Comment:
This is a very useful function, good for saving down a small image and keep the aspect ratio intact. I see no reason to remove it and to keep `resizeTo`.